src/pages/excursions.js

```diff
@@ -1,87 +1,84 @@
 import React from 'react';
-import Cabin from '../images/cabin.jpg'; 
+import Cabin from '../images/cabin.jpg';
 import '../App.css';
 
+const Excursions = () => {
+	return (
+		<div className='Container'
+			style={{
+				backgroundColor: '#B4BF5E',
+				marginTop: '-21px',
+				marginLeft: '282px',
+				marginRight: '282px',
+				height: '180vh'
+			}}>
+			<h1 className='about_h1' style={{
+				textAlign: "center",
+				paddingTop: 50
+			}}>Excursion 2021</h1>
 
 
-const Excursions = () => {
-return( 
-	<div className='Container'
-	style={{
-		backgroundColor: '#B4BF5E',
-		marginTop: '-21px',
-		marginLeft: '282px',
-		marginRight: '282px',
-		height: '180vh'
-	}}>
-	<h1 className='about_h1'style={{
-              textAlign: "center",
-			  paddingTop:50
-            }}>Excursion 2021</h1>
+			<div className='first-collumn-image'>
+				<img src={Cabin}
+					alt="sweden"
+					style={{
+						justifyContent: "center",
+						position: "absolute",
+						width: "50%",
+						height: "50%",
+						left: "402px",
+						top: "257px",
+						alignItems: 'center'
+					}}
+				/>
 
+			</div>
+			<div className='info'>
+				<p
+					style={{
+						fontWeight: 300,
+						fontFamily: "STIXVariants",
+						paddingTop: 500,
+						textAlign: "center",
+						fontSize: 28
+					}}
+				>
+					Friday to sunday<strong> June 7-9 2021</strong><br />Location: <strong>Vilshult in Småland, SWEDEN</strong><br />Price: Adult 950 kr.<br />  Child 475 kr.
```

Comment:
I get this is a work in progress, but I highly recommend putting all your "dummy"/test data in one place. a "testData.js" file for instance, and then import it. Hard-coding it like you do here gives you more work later on.